docs/content/latest/secure/tls-encryption/connect-to-cluster.md

```diff
@@ -19,16 +19,18 @@ To connect CLIs, tools, and APIs to a remote YugabyteDB cluster when client-to-s
 
 ## Prerequisites
 
-Before you can enable and use server-to-server encryption, you need to create and configure server certificates for each node of your YugabyteDB cluster. For information, see [Create client certificates](../client-certificates).
+Before you can use client certificates to connect to your YugabyteDB clusters, server-to-server encryption must be enabled. For details, see [Enable server-to-server encryption](../server-to-server).
 
-For each client that will connect to a YugabyteDB cluster, you need the following three files to be accessible on the client computer.
+Also, you need to create the required client certificates. For details, see [Create client certificates](../client-certificates).
 
-- `root.crt` — root certificate file
-  - To generate, see [Prepare root configuration file](../prepare-nodes/#generate-root-config)
+Each client that connects to a YugabyteDB cluster needs the following three files to be accessible on the client computer.
+
+- `ca.crt` — root certificate file
```

Comment:
Is this correct for YSQL? Or does YSQL filename need to be root.crt?

Reply:
Good question -- `root.crt` was my idea, but the code is looking for `ca.crt`.  I did a search and found two more instances with the wrong file name.